modules/standard/Time.chpl

```diff
@@ -24,7 +24,7 @@
 /* Support for routines related to measuring the passing of time.
 
    This module provides support for querying wall time in the local
-   timezone and implements a record :record:`~Timer` that provides basic
+   timezone and implements a record :record:`~stopwatch` that provides basic
    stopwatch behavior.  The stopwatch has the potential for microsecond
```

Comment:
This sounds slightly redundant. Maybe `implements a stopwatch that can measure the execution time of sections of a program` or something along those lines.